pkg/apis/kops/cluster.go

```diff
@@ -223,7 +223,8 @@ type ClusterSpec struct {
 
 // PodIdentityWebhookConfig configures an EKS Pod Identity Webhook.
 type PodIdentityWebhookConfig struct {
-	Enabled bool `json:"enabled,omitempty"`
+	Enabled  bool `json:"enabled,omitempty"`
+	Replicas int  `json:"replicas,omitempty"`
```

Comment:
I was wondering if it would make sense to set replicas to 0, but I guess someone that wanted to do that would set enabled = false instead.